docs/quickstart.html

```diff
@@ -359,7 +359,7 @@
 
 
 <pre>
-&gt; <b>cat file-input.txt | ./bin/kafka-console-producer --broker-list localhost:9092 --topic streams-file-input</b>
+&gt; <b>bin/kafka-console-producer.sh --broker-list localhost:9092 --topic streams-file-input < file-input.txt</b>
```

Comment:
This replacement had been made before to make the command work on both Linux and Windows systems.